README.md

```diff
@@ -125,7 +140,14 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+genie = Genie.create(name: "Genie")
+lamp = Lamp.create(wishes_remaining: 3)
+genie.lamp_id = lamp_id
```

Comment:
the `lamp_id` variable to the right of the equal sign isn't defined here... we could do `lamp.id` though!